### PR TITLE
Fixes #5 Fix humanize_bytes function to work properly

### LINKED ISSUE #5: Fix humanize_bytes function to work properly
@Skyvar  want to take a look at this?

https://github.com/Aftermath/neos/blob/master/neos/utils.py#L6

### PR DESCRIPTION
**Pull request by Aftermath**
Added a function to round the size to make it a little cleaner, but is perhaps unnecessary.
Maybe it can be re-used at some point.
What do you think @tchellomello, @Skyvar ?

### CODE REVIEW

**Comment by tchellomello** [comment]
@Aftermath  the patch looks good, but it has some issues with coding style. 

Once you address the issue #10, try to re-submit the PR to report the styles issues.

File: neos/utils.py [hunk 1]
```diff
@@ -3,10 +3,18 @@
 """Neos utils."""
 
 
-def humanize_bytes(value, unit='KB'):
-    """Return humanized value from kilobytes."""
+def round_size(value):
+    """Return value rounded to 2 decimal places"""
+    value=round(value, 2)
```
Comment (tchellomello) [inline neos/utils.py]: Make to have to have a space `value = round(value, 2)`

File: neos/utils.py [hunk 2]
```diff
@@ -3,10 +3,18 @@
...
+    return value
+
+def humanize_bytes(value, unit='B'):
+    """Return humanized value from bytes."""
+    if unit == 'KB':
+        value=value / 1024
```
Comment (tchellomello) [inline neos/utils.py]: Add the space after `value = `

File: neos/utils.py [hunk 3]
```diff
@@ -3,10 +3,18 @@
...
     if unit == 'MB':
-        return value / 1024
+        value=value / 1024 / 1024
     elif unit == 'GB':
-        return value / 1024 / 1024
-    return value
+        value=value / 1024 / 1024 / 1024
+    return round_size(value)
+
```
Comment (tchellomello) [inline neos/utils.py]: Remove the extra empty line.

**Review by tchellomello — CHANGES_REQUESTED** [review]
Good work @Aftermath :+1:

File: neos/utils.py [hunk 4]
```diff
@@ -3,10 +3,18 @@
 """Neos utils."""
 
 
-def humanize_bytes(value, unit='KB'):
-    """Return humanized value from kilobytes."""
+def round_size(value):
+    """Return value rounded to 2 decimal places"""
```
Comment (tchellomello) [inline neos/utils.py]: For coding sytle purposes, the docstring requires a period (.)

**Comment by Aftermath** [comment]
@tchellomello, do I need to recreate the entire PR, or just submit a new commit to that branch?

**Comment by tchellomello** [comment]
@Aftermath  you can use the same PR and same branch. Just update the code and update the branch with the `git push`. 

Now that you integrated travis and hound to the repository, some tests will be executed automatically.

File: neos/utils.py [hunk 5]
```diff
@@ -3,10 +3,17 @@
...
 
 
-def humanize_bytes(value, unit='KB'):
-    """Return humanized value from kilobytes."""
+def round_size(value):
+    """Return value rounded to 2 decimal places."""
+    value = round(value, 2)
+    return value
+
+def humanize_bytes(value, unit='B'):
```
[the remark this answers is not in the document]
Comment (Aftermath) [reply neos/utils.py]: I like this guy.

**Comment by Aftermath** [comment]
Looks like the Travis-CI builds are currently failing against the requirements its trying to pull in @tchellomello, but otherwise we are good!

**Pull request merged**